src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -46,6 +46,9 @@ gregtech.multiblock.fusion_reactor_mk2.description=The Fusion Reactor Mark 2 is
 gregtech.multiblock.fusion_reactor_mk3.description=The Fusion Reactor Mark 3 is a large multiblock structure used for fusing elements into heavier ones. The Mark 3 Fusion Reactor can only use UV Energy Hatches. For every Hatch it has its buffer increases by 40M EU.
 gregtech.multiblock.fusion_reactor.heat=Heat: %d
 gregtech.multiblock.large_chemical_reactor.description=The Large Chemical Reactor performs chemical reactions at 100% energy efficiency. Overclocks multiply both speed and energy by 4. The multiblock requires exactly 1 Cupronickel Coil Block, which must be placed adjacent to the PTFE Pipe casing located in the center.
+gregtech.multiblock.primitive_water_pump.description=The Primitive Water Pump is a pre-Steam Era multiblock that collects water once per second, depending on the Biome it is in. It can use a Pump, ULV, or LV Output Hatch, increasing the amount of water per tier. Collects up to 3000 L/s Water at max, and 100 L/s at minimum.
```

Comment:
Change the last sentence to say: "Collects up to 3000 L/s of Water at max [...]" instead of "Collects up to 3000 L/s Water at max [...]".

Should also mention how the multiplier affects the biome output.